pocs/focuser/focuser.py

```diff
@@ -46,6 +49,21 @@ def position(self, position):
         """ Move focusser to new encoder position """
         self.move_to(position)
 
+    @property
+    def camera(self):
+        """
+        Reference to the Camera object that the Focuser is assigned to, if any. A Focuser
+        should only ever be assigned to one or zero Cameras!
+        """
+        return self._camera
+
+    @camera.setter
+    def camera(self, camera):
+        if self._camera:
+            self.logger.error("{} already assigned to camera {}!".format(self, self.camera))
```

Comment:
OK. I made it an error as attempting to assign a `Focuser` instance to more than one `Camera` instance seemed like a pretty clear indication that something has gone wrong. I can't think of any situation where this would be what you intended to happen.